identifiers/country-ie/README.md

```diff
@@ -0,0 +1,6 @@
+# OCD Division Identifier
+
```

Comment:
Can you add a `Types` section listing out the various types used in this folder. For eg: https://github.com/opencivicdata/ocd-division-ids/blob/master/scripts/country-ca/README.md?plain=1#L23

Reply:
Followed your suggestion. Send an update. Please have a look.